_run/kube/deployment.yaml

```diff
@@ -52,4 +53,7 @@ deployment:
   bew:
     westcoast:
       profile: web
-      count: 1
\ No newline at end of file
+      count: 1
+endpoints:
+  foobario:
```

Comment:
_would_ `foobar.io` be a legal key?

Reply:
Probably, I can check. Do we want to allow names like that and other obscure things like "-_!" ?

Reply:
`-` and `_` would be nice.